tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -139,13 +186,22 @@ def _check_efs_after_nodes_reboot(all_mount_dirs, cluster, remote_command_execut
     for compute_node in compute_nodes:
         scheduler_commands.reboot_compute_node(compute_node, asap=False)
     scheduler_commands.wait_nodes_status("idle", compute_nodes)
-    _check_efs_correctly_mounted_and_shared(all_mount_dirs, remote_command_executor, scheduler_commands)
+    _check_efs_correctly_mounted_and_shared(
+        all_mount_dirs, remote_command_executor, scheduler_commands, iam_authorizations, encryption_in_transits
+    )
     return remote_command_executor, scheduler_commands
 
 
-def _check_efs_correctly_mounted_and_shared(all_mount_dirs, remote_command_executor, scheduler_commands):
-    for mount_dir in all_mount_dirs:
-        test_efs_correctly_mounted(remote_command_executor, mount_dir)
+def _check_efs_correctly_mounted_and_shared(
+    all_mount_dirs, remote_command_executor, scheduler_commands, iam_authorizations=None, encryption_in_transits=None
```

Comment:
Can we remove the None defaults?

Reply:
No, because using mutable types as defaults is bad practice.

Reply:
Can we not have the defaults at all? We do not need mutable types defaults, right?

Reply:
Yes, on closer inspection, every call of `_check_efs_correctly_mounted_and_shared` uses every argument. Furthermore, the calls are only initiated in `_check_efs_after_nodes_reboot` and `test_multiple_efs` and the only call to `_check_efs_after_nodes_reboot` is in `test_multiple_efs`. Finally, by `iam_authorizations` and `encryption_in_transits` being being defined in `test_multiple_efs` there is no scenario in which defaults would be necessary. So, I will remove them.